src/group_theory/commensurable.lean

```diff
@@ -0,0 +1,90 @@
+/-
+Copyright (c) 2021 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import group_theory.index
+import group_theory.quotient_group
+import group_theory.subgroup.pointwise
+import group_theory.group_action.conj_act
+/-!
+# Commensurability for subgroups
+
+This file defines commensurability for subgroups of a group `G`. It then goes on to prove that
+commensurability defines an equivalence relation and finally defines the commensurator of a subgroup
+of `G`.
+
+## Main definitions
+
+* `commensurable`: defines commensurability for two subgroups `H`, `K` of  `G`
+* `commensurator`: defines the commensurator of a subgroup `H` of `G`.
```

Comment:
Can you explain somewhere the difference between `commensurator'` and `commensurator`? In a section like `## Implementaton details` or something like that. If `commensurator'` is the main definition, it should be called `commensurator` I think, and if `commensurator` is the main definition you should add it here.